nansat/mappers/mapper_netcdf_cf.py

```diff
@@ -194,7 +194,7 @@ class BreakI(Exception):
                 # Keep only desired bands (given in "bands" list)
                 try:
                     if bands:
-                        if not band_metadata.has_key('standard_name'):
+                        if 'standard_name' not in band_metadata.keys():
```

Comment:
what's the reason for this change? is it faster?

Reply:
method has_key was deprecated in Python 3